src/state.c

```diff
@@ -1305,10 +1304,11 @@ doDetectFile(const char *path, fs_info *fs, struct stat *sbuf)
     // check for spaces at the end of file names
     for (i = 0; i < scope_strlen(path); i++) {
         if (scope_isspace(path[i])) {
-            char msg[PATH_MAX + 128];
+            char msg[REASON_MAX];
 
             scope_snprintf(msg, sizeof(msg), "spaces in the path name %s representing a potential issue",
                            path);
+            fileSecurity(path, msg, FALSE, 0);
```

Comment:
The msg here assumes the string "Process %s (pid %d) on host %s encountered %s"  in front of msg. Where the last var is the msg. Some of these msg strings may not make sense in a sec event? Maybe it's OK because they are formatted as an event?

Reply:
I think it should be ok since we are formatting as an event